client/api/openapi.yaml

```diff
@@ -6423,6 +6423,25 @@ components:
             $ref: '#/components/schemas/NodePoolsAmazon'
           type: object
       type: object
+    CreateEKSProperties_eks_master:
```

Comment:
Please remove the `master` section from create EKS cluster openapi spec as for EKS this is not valid. This seems to be an oversight from a previous commit.

Reply:
Will be done in a separate PR